app/controllers/api/v3/v3_application_controller.rb

```diff
@@ -0,0 +1,19 @@
+module Api
+  module V3
+    class V3ApplicationController < ApplicationController
```

Comment:
is this V3 prefix needed in the class name?

Reply:
Most likely it's not, I'll remove it